esmf_regrid/tests/unit/experimental/io/test_round_tripping.py

```diff
@@ -548,3 +548,18 @@ def test_generic_regridder_resolution(tmp_path, scheme):
     assert loaded_rg.regridder.src.resolution == src_resolution
     assert loaded_rg.tgt_resolution == tgt_resolution
     assert loaded_rg.regridder.tgt.resolution == tgt_resolution
+
+
+def test_precomputed(tmp_path):
+    """Test save/load round tripping for regridders made with precomputed weights."""
+    original_rg, src = _make_grid_to_mesh_regridder(
+        method=None, regridder=ESMFAreaWeightedRegridder
+    )
+    tgt = original_rg._tgt
+    weights = original_rg.regridder.weight_matrix
+    original_rg = ESMFAreaWeightedRegridder(src, tgt, precomputed_weights=weights)
+    assert original_rg.regridder.esmf_version == "N/A"
```

Comment:
@stephenworsley Can you make this `esmf_regrid.esmg_regridder.ESMF_NO_VERSION` instead?

That way it continues to work independent of the value 👍